test/java/org/chromium/net/testing/MetricsTestUtil.java

```diff
@@ -0,0 +1,170 @@
+package org.chromium.net.testing;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import android.os.ConditionVariable;
+import java.util.Date;
+import java.util.LinkedList;
+import java.util.NoSuchElementException;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+import org.chromium.net.RequestFinishedInfo;
+import org.chromium.net.RequestFinishedInfo.Listener;
+
+/**
+ * Classes which are useful for testing Cronet's metrics implementation and are needed in more than
+ * one test file.
+ */
+public class MetricsTestUtil {
```

Comment:
nit: please mark as "final", and add a private constructor at the end of the class.